iot-hub/Samples/service/JobsSample/Parameters.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using CommandLine;
+
+namespace Microsoft.Azure.Devices.Samples.JobsSample
+{
+    /// <summary>
+    /// Parameters for the application.
+    /// </summary>
+    internal class Parameters
+    {
+        [Option(
+            'p',
+            "HubConnectionString",
+            Required = false,
+            HelpText = "The connection string of the IoT hub instance to connect to. This can be located under \"Shared Access Policies\" in the Iot hub.")]
```

Comment:
You can move the "This can be located under..." up to class remarks, per my PR.

https://github.com/Azure-Samples/azure-iot-samples-csharp/pull/316

Reply:
Also, add a note that it defaults to the env var, by name. Also demonstrated in that PR.